VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-status/participant-status.component.html

```diff
@@ -4,7 +4,20 @@
 
 <div id="participant-status-list">
   <div *ngFor="let participant of participants" class="govuk-grid-row">
-    <div class="govuk-grid-column-one-half">
+    <div class="govuk-grid-column-one-half status-box-display">
+      <div class="interpreter-icon">
```

Comment:
judge icon also in this container.. should the name of this class be changed?

Reply:
changed to participant-icon